### PR TITLE
Add anonymize and create_histograms parameters to metadata rebuild endpoints

### PR DESCRIPTION
**Pull request by the-bird-is-the-word**
Adds anonymize and create_histograms parameters to metadata rebuild endpoints. These can be used to anonymize data when uploading data or after data has already been uploaded.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ## [major.minor.patch] - DD-MM-YYYY
 
+## [2.2.0] - 14.11.2024
+
+- added support for `anonymize` and `create_histograms` parameters to `trigger_metadata_rebuild_job` and `trigger_dataset_metadata_rebuild_job`
```
[the remark this answers is not in the document]
Comment (the-bird-is-the-word) [reply CHANGELOG.md:5]: Yes, I will add it :)

File: setup.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 setup(
     name="hari_client",
-    version="2.0.3",
+    version="2.2.0",
```
[the remark this answers is not in the document]
Comment (katia-qm) [reply setup.py:6]: Yes, it was indeed not released, good catch! @the-bird-is-the-word could you include it in 2.1.0 then?

[the remark this answers is not in the document]
Comment (the-bird-is-the-word) [reply setup.py:6]: Ah yes, then let us add it to 2.1.0. I will update it :)

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,11 @@
 ## [major.minor.patch] - DD-MM-YYYY
 
+## [2.2.0] - 14.11.2024
+
+### New Features
+
+- added support for `anonymize` and `calculate_histograms` parameters to `trigger_metadata_rebuild_job` and `trigger_dataset_metadata_rebuild_job`
+
```
[the remark this answers is not in the document]
Comment (the-bird-is-the-word) [reply CHANGELOG.md]: Ah sorry, forgot about the changelog :)

**Pull request merged**